### ISSUE TITLE
Shadow only draws if root child view has a background

### ISSUE THREAD

**Issue by danhawkes** [h6ah4i/android-materialshadowninepatch#3]
Is this by design?

I have something like this:

```
<MaterialShadowContainerView>
   <LinearLayout>
    <View android:background="#ff0000">
    <View android:background="#00ff00">
  </LinearLayout>
</MaterialShadowContainerView>
```

The shadow does not seem to draw if the LinearLayout does not have a background. If I set one, it works, but also means there's unnecessary overdraw. 

Thanks for the very useful library, by the way!

**Comment by h6ah4i** [comment]
Hi. Sorry for the very late response :disappointed: 

That behavior is as expected because Lollipop's drop shadow also requires solid color background.

Related info:
- [Elevation on Android Lollipop not working](http://stackoverflow.com/questions/26572048/elevation-on-android-lollipop-not-working)

**Comment by h6ah4i** [comment]
Over draws issue has been fixed in 0.6.2.

Specify `style="@style/ms9_CompositeShadowStyle"` if you want to reduce overdraws.
